api/swagger/swagger.yaml

```diff
@@ -229,6 +229,27 @@ paths:
           description: Error
           schema:
             $ref: "#/definitions/ErrorResponse"
+  /restaurant:
+    x-swagger-router-controller: PromotionController
+    post:
+      description: Uploads csv file of restaurant/promotion info, admin required.
+      operationId: createPromotion
+      consumes:
+        - multipart/form-data
+      parameters:
+        - name: Authorization
+          in: header
+          type: string
+          required: true
+        - name: upload
+          in: formData
+          type: file
+          required: true
+      responses:
+        default:
+          description: Error
+          schema:
+            $ref: "#/definitions/ErrorResponse"
```

Comment:
Should we also put a success code here?
 -- The future tests might need the success code.
"204":
          description: Success